src/OAuth2/Storage/JwtAccessToken.php

```diff
@@ -34,11 +34,11 @@ public function __construct(PublicKeyInterface $publicKeyStorage, AccessTokenInt
     public function getAccessToken($oauth_token)
     {
         // just decode the token, don't verify
-        if (!$tokenData = $this->encryptionUtil->decode($oauth_token, null, false)) {
+        if (!$tokenData = $this->tokenStorage->getAccessToken($oauth_token)) {
```

Comment:
Why does this need to be changed in order to support the latest version of `firebase/php-jwt`?

Reply:
It is some time ago, I do not really remember. But I think there was a problem that the newest firebase needed the key always, and in this if-test, `null` was passed.

The idea here is to decode the token data passed, and in my project at least, the tokenStorage had all information I needed. But maybe you should do it in 2 steps then, first get data from tokenStorage (I think it contains the key) and then decode with the encryptionUtil.

But if I don't remember wrong, I think it is not possible to pass `null` to the decode method if firebase is used.